Audio/WaveBankReader.cpp

```diff
@@ -229,7 +224,7 @@ namespace
         DWORD AdpcmSamplesPerBlock() const
         {
             uint32_t nBlockAlign = (wBlockAlign + ADPCM_BLOCKALIGN_CONVERSION_OFFSET) * nChannels;
-            return nBlockAlign * 2 / (uint32_t)nChannels - 12;
+            return nBlockAlign * 2 / uint32_t(nChannels) - 12;
```

Comment:
``-Wold-style-casts`` in clang found a lot of these lingering in the code, so I fixed them.